test/plugins/node-workspace.ts

```diff
@@ -372,6 +376,171 @@ describe('NodeWorkspace plugin', () => {
+      const updates = nodeCandidate!.pullRequest.updates;
+      assertHasVersionUpdate(updates, 'node1/package.json', '3.3.4');
+      assertHasVersionUpdate(updates, 'node2/package.json', '2.2.3');
+      assertHasVersionUpdate(updates, 'node3/package.json', '1.1.2');
+      snapshot(dateSafe(nodeCandidate!.pullRequest.body.toString()));
+      const update = assertHasUpdate(updates, 'node1/CHANGELOG.md', Changelog);
+      snapshot((update.updater as Changelog).changelogEntry);
+      const compositeUpdater = assertHasUpdate(
+        updates,
+        'node2/CHANGELOG.md',
+        CompositeUpdater
+      ).updater as CompositeUpdater;
+      for (const changelogUpdater of compositeUpdater.updaters) {
+        snapshot((changelogUpdater as Changelog).changelogEntry);
+      }
+      const compositeUpdater2 = assertHasUpdate(
+        updates,
+        'node3/CHANGELOG.md',
+        CompositeUpdater
+      ).updater as CompositeUpdater;
+      for (const changelogUpdater of compositeUpdater2.updaters) {
+        snapshot((changelogUpdater as Changelog).changelogEntry);
+      }
+    });
+    it('incorporates extra-files from strategy', async () => {
+      const candidates: CandidateReleasePullRequest[] = [
+        buildMockCandidatePullRequest('node1', 'node', '3.3.4', {
+          component: '@here/pkgA',
+          updates: [
+            buildMockPackageUpdate('node1/package.json', 'node1/package.json'),
+            buildMockChangelogUpdate(
+              'node1/CHANGELOG.md',
+              '3.3.4',
+              'other notes'
+            ),
+          ],
+        }),
+      ];
+      stubFilesFromFixtures({
+        sandbox,
+        github,
+        fixturePath: fixturesPath,
+        files: [
+          'node1/package.json',
+          'node2/package.json',
+          'node3/package.json',
+          'node4/package.json',
+          'node5/package.json',
+        ],
+        flatten: false,
+        targetBranch: 'main',
+      });
+      await plugin.preconfigure(
+        {
+          node1: new Node({
+            github,
+            targetBranch: 'main',
+            path: 'node1',
+            packageName: '@here/pkgA',
+          }),
+          node2: new Node({
+            github,
+            targetBranch: 'main',
+            path: 'node2',
+            packageName: '@here/pkgB',
+            extraFiles: ['my-file'],
+          }),
+        },
+        {},
+        {
+          node2: {
+            tag: new TagName(new Version(2, 2, 2), 'pkgB'),
+            sha: '',
+            notes: '',
+          },
+        }
+      );
+
+      const newCandidates = await plugin.run(candidates);
+      expect(newCandidates).lengthOf(1);
+      const nodeCandidate = newCandidates.find(
+        candidate => candidate.config.releaseType === 'node'
+      );
+      expect(nodeCandidate).to.not.be.undefined;
+      const updates = nodeCandidate!.pullRequest.updates;
+
+      assertHasUpdate(updates, 'node2/my-file', Generic);
```

Comment:
Without invoking the base strategy we would not have the update for extra-files.